Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM php:7.1-cli
+RUN curl https://getcomposer.org/download/1.5.2/composer.phar -o /usr/local/bin/composer && chmod +x /usr/local/bin/composer
```

Comment:
Could use https://hub.docker.com/r/library/composer/?

Reply:
This is an image built only for running Composer, so it could have a place in a multi-stage build of a PHP project, but it is a bit too complex for now